recipes/yasm/all/conandata.yml

```diff
@@ -4,3 +4,6 @@ sources:
       sha256: "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
     - url: "https://raw.githubusercontent.com/yasm/yasm/bcc01c59d8196f857989e6ae718458c296ca20e3/YASM-VERSION-GEN.bat"
       sha256: "b976cb97d2f7bb00e78e5db0da0978659acbdd60b52998cd2983145d8d75f141"
+patches:
+  "1.3.0":
+    - patch_file: "patches/0001-platformtoolset.patch"
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches-fields

Reply:
Not sure if the fields are ok.